assets/helpers/abTests/abtestDefinitions.js

```diff
@@ -4,16 +4,4 @@ import type { Tests } from './abtest';
 // ----- Tests ----- //
 
 export const tests: Tests = {
-  directDebitTest: {
-    variants: ['control', 'directDebit'],
-    audiences: {
-      GB: {
-        offset: 0,
-        size: 1,
-      },
-    },
-    isActive: true,
```

Comment:
I would put `isActive: false` so in that way we remember that we still need to remove the test from the code.